### ISSUE TITLE
Devmenu

### ISSUE THREAD

**Issue by Zilverfire** [Reisyukaku/ReiNX#54]
Can't boot devmenu (via installed to Pokemon quest in the titles for Reinx) it gets stuck on the nintendo branded loading screen then you have to shutdown

**Comment by Reisyukaku** [comment]
Sounds like your files.

**Comment by jkid291** [comment]
I have been using the devmenu pokemon quest on 1.2 just fine, till 1.6. I know I can use tinfoil for nsp's but I like to use devmenu for its other features, is there a new sd card file pack I need for 1.6? I just installed the what was on your release but i still have everything for 1.2 (minus the files with the same name, I just extracted over the old files deleting the old ones leaving the new ones in their place, do I need to just do a fresh install of reinx and my other files to the sd card?
